nixos/modules/services/x11/redshift.nix

```diff
@@ -18,73 +18,14 @@ in {
       '';
     };
 
-    latitude = mkOption {
-      type = types.nullOr types.str;
-      default = null;
+    config = mkOption {
+      type = types.attrs;
+      default = {};
       description = ''
-        Your current latitude, between
-        <literal>-90.0</literal> and <literal>90.0</literal>. Must be provided
-        along with longitude.
+        Configuration to give to redshift.
```

Comment:
The description should be a lot longer, explain what this option does and how it can be used.

Reply:
Referring to man page now, I hope that's sufficient.